src/integrators/gpt/gpt.cpp

```diff
@@ -1198,15 +1202,15 @@ GradientPathIntegrator::GradientPathIntegrator(const Properties &props)
 
 	if(m_config.m_reconstructAlpha <= 0.0f)
 		Log(EError, "'reconstructAlpha' must be set to a value greater than zero!");
+
+	if (m_config.m_maxDepth < -1)

```

Comment:
This should actually be 'if (m_config.maxDepth <= 0 && m_config.maxDepth != -1)' to also block zero